src/main/java/seedu/address/model/leave/Leave.java

```diff
@@ -65,7 +66,7 @@ public Leave(Person employee, String title, LocalDate start, LocalDate end)
         this.description = "";
         this.start = start;
         this.end = end;
-        this.status = "Pending";
+        this.status = Status.of(StatusType.PENDING.toString());
```

Comment:
If the default status is PENDING, we can create a default constructor for Status that does not take in any parameters, and returns a Status object with a value of PENDING.